src/sage/features/sagemath.py

```diff
@@ -329,7 +431,31 @@ def __init__(self):
             sage: isinstance(sage__rings__real_mpfr(), sage__rings__real_mpfr)
             True
         """
-        PythonModule.__init__(self, 'sage.rings.real_mpfr')
+        PythonModule.__init__(self, 'sage.rings.real_mpfr',
+                              spkg='sagemath_modules')
+
+
+class sage__schemes(JoinFeature):
+    r"""
+    A :class:`~sage.features.Feature` describing the presence of :mod:`sage.schemes`.
+
+    EXAMPLES::
+
+        sage: from sage.features.sagemath import sage__schemes
+        sage: sage__schemes().is_present()  # optional - sage.schemes
```

Comment:
shouldn't we have more spaces before the `# optional` tag ?

Reply:
Thanks. Done in 3c002acf7b; the same for the existing features in this file is taken care of in:
- #35749